src/components/RoundedBackground.tsx

```diff
@@ -1,13 +1,13 @@
 import { heightPercentageToDP as hp } from 'react-native-responsive-screen'
 import styled from 'styled-components/native'
 
-const RoundedBackground = styled.View<{ color?: string; height?: string }>`
+const RoundedBackground = styled.View<{ color?: string; height?: string; marginBottom?: string }>`
   width: 140%;
   height: ${props => props.height ?? '50%'};
   background-color: ${props => props.color ?? props.theme.colors.primary};
   border-bottom-left-radius: ${hp('60%')}px;
   border-bottom-right-radius: ${hp('60%')}px;
-  margin-bottom: ${props => props.theme.spacings.xxl};
+  margin-bottom: ${props => props.marginBottom ?? props.theme.spacings.xxl};
```

Comment:
This can be reverted as well since you only ever pass `marginBottom={theme.spacings.xxl}` which is the default anyway: `margin-bottom: ${props => props.marginBottom ?? props.theme.spacings.xxl};`